packages/twenty-front/src/modules/settings/page-layout/hooks/usePageLayoutActiveTabId.ts

```diff
@@ -0,0 +1,19 @@
+import { SETTINGS_PAGE_LAYOUT_TABS_INSTANCE_ID } from '@/settings/page-layout/constants/SettingsPageLayoutTabsInstanceId';
+import { activeTabIdComponentState } from '@/ui/layout/tab-list/states/activeTabIdComponentState';
+import { useRecoilComponentValue } from '@/ui/utilities/state/component-state/hooks/useRecoilComponentValue';
+import { useSetRecoilState } from 'recoil';
+
+export const usePageLayoutActiveTabId = () => {
```

Comment:
this is not really useful, just exposing states

States are actually part of the API and can be called directly
+ this hook is exposing a reactive value + a non reactive sync function, if we want to use the sync function, we will have to listen on the reactive value which is not great